deepchem/molnet/load_function/swissprot_datasets.py

```diff
@@ -0,0 +1,79 @@
+"""
+swissprot dataset loader.
+"""
+import os
+import deepchem as dc
+from deepchem.molnet.load_function.molnet_loader import TransformerGenerator, _MolnetLoader
+from deepchem.data import Dataset
+from typing import List, Optional, Tuple, Union
+
+SWISSP_URL = "https://deepchemdata.s3-us-west-1.amazonaws.com/datasets/uniprot_swissprot_14_06_21.csv"
+SWISSP_TASK=[]
+
+class _SWISSPROTLoader(_MolnetLoader):
+  
+  def create_dataset(self) -> Dataset:
+    dataset_file = os.path.join(self.data_dir, "uniprot_swissprot_14_06_21.csv")
+    if not os.path.exists(dataset_file):
+      dc.utils.data_utils.download_url(url=SWISSP_URL, dest_dir=self.data_dir)
+      dataset_file = os.path.join(self.data_dir, "uniprot_swissprot_14_06_21.csv")
+
+      
+    loader = dc.data.CSVLoader(
+        tasks=self.tasks, featurizer=self.featurizer, sanitize=True)
+    return loader.create_dataset(dataset_file, shard_size=8192)
+
+  codes = ['A', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'K', 'L',
+         'M', 'N', 'P', 'Q', 'R', 'S', 'T', 'V', 'W', 'Y']
+         
+  def load_swissprot(
+    featurizer: Union[dc.feat.Featurizer, str] = dc.feat.OneHotFeaturizer(codes),
+    splitter: Union[dc.splits.Splitter, str, None] = 'random',
+    transformers: List[Union[TransformerGenerator, str]] = [],
+    reload: bool = True,
+    data_dir: Optional[str] = None,
+    save_dir: Optional[str] = None,
+    **kwargs
+) -> Tuple[List[str], Tuple[Dataset, ...] ]:
+    """Load Swissprot dataset.
+
+    The Swiss-Prot Database is parto og the UniprotKnowledebase (UniprotKB) that contains a
+    collection of functiona information on protiens. Specially the Swiss-Prot Database 
+    contains records with information extracted from literature and curator-evaluated computational
```

Comment:
Can you add some lines below this paragraph that describe exactly what information is returned in the dataset? Protein sequence, properties, etc. And also how many sequences are in the dataset.

Reply:
Done ! For now is a small description. Only contain the sequences, but it would be really cool to add new properties.